components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth/config/OAuthServerConfiguration.java

```diff
@@ -1777,12 +1780,14 @@ private void parseSupportedResponseTypesConfig(OMElement oauthConfigElem) {
             // if this element is not present, assume the default case.
             log.warn("\'SupportedResponseTypes\' element not configured in identity.xml. " +
                     "Therefore instantiating default response type handlers");
-
             Map<String, String> defaultResponseTypes = new HashMap<>(4);
```

Comment:
We are initializing a map of 4 elements and adding more than four elements. Why define the size at all?